ccdproc/image_collection.py

```diff
@@ -58,14 +58,19 @@ class ImageFileCollection(object):
         The filenames are assumed to be in ``location``.
         Default is ``None``.
 
+     ext: str or int, optional
+         Name/Index of the extension, which specifies the extension to extract data
```

Comment:
Actually, I fell asleep while writing that, and when I woke up, I forgot and pushed it anyway :)
Correcting it now.